lib/ansible/modules/network/ios/ios_ntp.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+DOCUMENTATION = '''
+---
+module: ios_ntp
+extends_documentation_fragment: ios
+version_added: "2.8"
+short_description: Manages core NTP configuration.
+description:
+    - Manages core NTP configuration.
+author:
+    - Federico Olivieri (@Federico87)
+options:
+    server:
+        description:
+            - Network address of NTP server.
+    source_int:
+        description:
+            - Source interface for NTP packets.
+    acl:
+        description:
+            - ACL for peer/server access restricition.
+    logging:
+        description:
+            - Enable NTP logs. Data type boolean.
+        type: bool
+        default: False
+        choices: [True, False]
```

Comment:
`choices` is unnecessary with `type: bool`

Reply:
Done